scripts/post.lua

```diff
@@ -1,17 +1,24 @@
 local pipelineDepth = 1
 
+function readData(filename)
+   local f = io.open(filename, "rb")
+   local data = f:read("*a")
+   f:close()
+   return data
+end
+
 function init(args)
    wrk.method = "POST"
-   wrk.body   = "foo=bar&baz=quux"
-   wrk.headers["Content-Type"] = "application/x-www-form-urlencoded"
+   wrk.body = readData("scripts/data.txt")
+   wrk.headers["Content-Type"] = "text/plain"
```

Comment:
Please confirm this file is presently unused. Might otherwise affect existing scenarios.